tests/funcadelic-test.js

```diff
@@ -203,4 +201,4 @@ describe('stable function', () => {
       expect(stable(stabilized)).to.equal(stabilized);
     });
   });
-});
+});
```

Comment:
We should avoid this kind of editor churn with some linting rules.

Reply:
I'll add eslint and jest in another PR.